schemas/utilities/occurrence_type_identifier.xml

```diff
@@ -3,9 +3,13 @@
 <header>List of possible occurrence types</header>
 <ddInstance xpath="ids_properties/occurrence_type"/>
 
-<!-- So far, we have defined an initial occurrence type list for the Equilibrium IDS only -->
+<!-- Initial occurrence type list for the Equilibrium IDS only -->
 <int name="reconstruction"  description="Equilibrium reconstruction">1</int>
 <int name="prediction_fixed"  description="Equilibrium prediction, fixed boundary">2</int>
 <int name="prediction_free"  description="Equilibrium prediction, free boundary">3</int>
 <int name="mapping"  description="Used for mapping equilibrium results from one grid type / resolution to another, or for including variables not present in the first set such as the calculation of magnetic field of other derived parameters">4</int>
+<!-- To distinguish between experimental and simulated data -->
```

Comment:
What is the recommendation for creators?  Should they use the more specific versions above?  Can any of the more specific cases fall into both "experiment" and "simulation".
To be more specific, are there cases when you might want one of the more specific labels from above and the more generic values below?